cloud/sts.go

```diff
@@ -0,0 +1,76 @@
+package cloud
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/pkg/errors"
+)
+
+type STSManager interface {
```

Comment:
nit: doc comment for what this is for (+ other exported funcs/structs in this file)